setup.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/sh
+CURRENT_DIR="$(cd "$(dirname "${0}")" && pwd)"
+
+
+cd "${CURRENT_DIR}"
+SECS="3"
+
+case ${1} in
+    mysql)
+        DB_ENG="mysql"
+        DB_HOST="mysql:3306"
+        ;;
+    postgres*)
+        DB_ENG="pgsql"
+        DB_HOST="postgres:5432"
+        ;;
+    *)
+        >&2 echo "Choose one of: mysql, postgres"
+        exit 1
+        ;;
+esac
+
+if [ $# -gt 2 ] && [ -f ${2} ]; then
+    set -o allexport
```

Comment:
what is this for?, could you add a #comment describing why it's required?

Reply:
It's being use as a shorthand to export the variables defined in the file passed as a .env file.
```suggestion
    # export all variables in env file
    set -o allexport
```